src/pages/Home.jsx

```diff
@@ -18,11 +19,7 @@ const Home = () => {
 
   return (
     <>
-      <h1>REACT TEMPLATE</h1>
-      <Button handleClick={() => dispatch(setExample("changing redux state"))}>
-        Click me
-      </Button>
-      <p>{example}</p>
+      <Categories />
```

Comment:
You shoud call it CategoryThumbNail or Category not Categories. It is only one category
It should receive as props an image url and the text should come as a child to use the composition pattern like so:
```
<Category imgUrl="dsdgsdggds>
Sports
</Category>
```
This way it is truly reusable and scalable